### ISSUE TITLE
Login page does not accept locales (Version 1.1.1)

### ISSUE THREAD

**Issue by modreoci** [solidusio-contrib/solidus_i18n#10]
It displays still in the default language.
![bw_login_page](https://cloud.githubusercontent.com/assets/10616435/12462643/b74183d6-bfbf-11e5-8660-c13967aee4bb.png)

**Comment by tvdeyen** [comment]
Do you have `SolidusI18n ::Config.available_locales` configured to include `sk`?

https://github.com/solidusio-contrib/solidus_i18n/blob/master/lib/solidus_i18n/controller_locale_helper.rb#L19

**Comment by modreoci** [comment]
I've got this:
```ruby
SolidusI18n::Config.available_locales = [:en, :sk] # displayed on frontend select box
SolidusGlobalize::Config.supported_locales = [:en, :sk] # displayed on translation forms
```
In my *spree.rb* file

Rest of the web work fine (meaning locales). There are a few more bugs and malfunctions out there (I'm little bit vexed by it).
Try to see it in http://bwshop.herokuapp.com/
I can provide you the admin account if you want.
It operates near the top of memory quota so it can freeze suddenly...

**Comment by tvdeyen** [comment]
Ok, thanks. I guess the problem is that the login controller (`Spree::UserSessionsController`) does not inherit from `Spree::BaseController` where we include the `SolidusI18n::ControllerLocaleHelper`.

We should also `include SolidusI18n::ControllerLocaleHelper` in `Spree::UserSessionsController`.

I will provide a pull request that fixes that. Meanwhile you can fix this by manually adding this into your app:

```ruby
# config/initializers/spree_i18n.rb
Spree::UserSessionsController.class_eval do
  include SolidusI18n::ControllerLocaleHelper
end
```

**Comment by modreoci** [comment]
I've got no such file in my app. Can I create the new one?

**Comment by tvdeyen** [comment]
Yes, please.

**Comment by modreoci** [comment]
Oki... It works! But... the same situation at http://localhost:3000/sk/admin/login :-)

**Comment by tvdeyen** [comment]
Yes, there are much more controllers that need to be patched. They should inherit from `Spree::BaseController`. Will open an issue on that repo as well.

**Comment by modreoci** [comment]
Me, you or someone else? :-)

**Comment by tvdeyen** [comment]
I will do.

**Comment by modreoci** [comment]
Can you inform me then?

**Comment by tvdeyen** [comment]
No. 1/2:

* https://github.com/solidusio/solidus/pull/727

**Comment by modreoci** [comment]
I'm not quite sure if I properly understand... Partly due to my english and partly due to my poor coding skills. 
Is it necessary to let to run the `solidus_auth_devise` command?

**Comment by tvdeyen** [comment]
@modreoci just a reminder, that this is still in progress. Currently you can't do anything but wait, sorry.

**Comment by AndreiMotinga** [comment]
What's the status on this one? I can't get the locale form to show up on frontend
